dashboard/app/models/lesson.rb

```diff
@@ -181,9 +181,9 @@ def render_property(property_name)
   # page, and the lesson plan pdf as a backup.
   def start_url
     if script_levels.first
-      return url_from_path(build_script_level_path(script_levels.first))
+      return root_url.chomp('/') + build_script_level_path(script_levels.first)
```

Comment:
here is one idea:
```suggestion
      return "https:#{url_from_path(build_script_level_path(script_levels.first))}"
```
another would be to add a new helper method with gets a url with protocol. 

the fact that we have unit tests covering this make me feel good about not getting into a bad state later if url_from_path ever changes (e.g. if we try to move entirely onto https urls within url_from_path).

Reply:
It turns out the underlying method `CDO.studio_url` takes a second parameter `scheme` so I was able to leverage that to specify `https`. The downside is that this link won't work locally. I took another look at the discussion on the last PR since I understand the problem a bit better now, it does seem surprising that there isn't an easy way to match the protocol for the links on the page to the protocol for the page.. that would be ideal